packages/terra-button/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changelog
 Unreleased
 ----------
 ### Changed
+* Inline SVG aligned for IE and wdio screenshots Updated.
+* Import for package version from package.json added to doc page
```

Comment:
This line seem unrelated to the changes in this package.

Reply:
Resolved in 2ff8a50